package.json

```diff
@@ -29,25 +29,14 @@
       "url": "http://www.opensource.org/licenses/MIT"
     }
   ],
-  "main": "./app/main.js",
+  "main": "./src/main/main.js",
   "scripts": {
-    "start": "gulp && ./node_modules/.bin/electron-forge start -- dev",
```

Comment:
we need to bring back a lot of things here - probably we should make another branch for tooling / scripts